MonitorControl/Display.swift

```diff
@@ -121,35 +116,38 @@ class Display {
 
     let isAlreadySet = volumeOSDValue == self.getValue(for: .audioSpeakerVolume)
 
-    if !isAlreadySet {
-      guard self.ddc?.write(command: .audioSpeakerVolume, value: volumeDDCValue) == true else {
-        return
+    DispatchQueue.global(qos: .userInitiated).async {
```

Comment:
This will undo the performance improvements for repeated and held-down media key volume commands made by executing the set volume DDC command on the main thread (per my initial analysis in #134).

Reply:
Are you saying we should be using `DispatchQueue.main` instead of  `DispatchQueue.global(qos: .userInitiated)` here?

Reply:
The DDC write command is already being run on the main thread, so you shouldn't need to enclose this in a `DispatchQueue` block at all, unless you're running the `setVolume` method from a background thread.

I'm happy to test out this PR and tinker with it as well to see if there's any regression in performance from this change though - my monitor at work has an OSD that behaves in a similar way to yours, but my home one doesn't, so I only have limited opportunities to test.

Reply:
> you shouldn't need to enclose this

Doesn't hurt to be explicit, especially since in this case it is critical where it is run.

> I'm happy to test out this PR

Please do.